### PR TITLE
Convert journald configuration instructions to Butane

### PR DESCRIPTION
**Pull request by bgilbert**
Avoid separately applying base64 to input files.

cc @bobfuru @miabbott

### CODE REVIEW

File: modules/cluster-logging-systemd-scaling.adoc [hunk 1]
```diff
@@ -17,45 +17,64 @@ and other settings.
...
 * `ForwardToConsole` to forward logs to the system console.
 * `ForwardToKsmg` to forward logs to the kernel log buffer.
 * `ForwardToSyslog` to forward to a syslog daemon.
 * `ForwardToWall` to forward messages as wall messages to all logged-in users.
-<3> Specify the maximum time to store journal entries. Enter a number to specify seconds. Or
+<4> Specify the maximum time to store journal entries. Enter a number to specify seconds. Or
 include a unit: "year", "month", "week", "day", "h" or "m". Enter `0` to disable. The default is `1month`.
-<4> Configure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst`
+<5> Configure rate limiting. If, during the time interval defined by `RateLimitIntervalSec`, more logs than specified in `RateLimitBurst`
 are received, all further messages within the interval are dropped until the interval is over. It is recommended to set
```
Comment (bobfuru) [inline modules/cluster-logging-systemd-scaling.adoc:66]:
```suggestion
<5> Configure rate limiting. If more logs are received than what is specified in `RateLimitBurst` during the time interval defined by `RateLimitIntervalSec`, all further messages within the interval are dropped until the interval is over. It is recommended to set
```
The sentence is kind of clunky, so suggesting a minor revision for clarity if the meaning is preserved.

File: modules/cluster-logging-systemd-scaling.adoc [hunk 2]
```diff
@@ -17,45 +17,64 @@ and other settings.
...
 `RateLimitIntervalSec=30s` and `RateLimitBurst=10000`, which are the defaults.
-<5> Specify how logs are stored. The default is `persistent`:
+<6> Specify how logs are stored. The default is `persistent`:
 * `volatile` to store logs in memory in `/var/log/journal/`.
 * `persistent` to store logs to disk  in `/var/log/journal/`. systemd creates the directory if it does not exist.
 * `auto` to store logs in  in `/var/log/journal/` if the directory exists. If it does not exist, systemd temporarily stores logs in `/run/systemd/journal`.
```
Comment (bobfuru) [inline modules/cluster-logging-systemd-scaling.adoc:71]: Remove extra " in"

**Review by bobfuru — COMMENTED** [review]
A couple of minor nits and otherwise this LGTM. 🤜 🤛

**Comment by bobfuru** [comment]
FYI @mburke5678 (because of https://github.com/openshift/openshift-docs/pull/26243) that this is making some journald config changes

**Comment by bgilbert** [comment]
Updated.

**Review by miabbott — APPROVED** [review]
LGTM

**Pull request merged**